eth/backend.go

```diff
@@ -229,11 +233,20 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 
 	eth.bloomIndexer.Start(eth.blockchain)
 
+	if config.BlobPool.Datadir != "" {
+		config.BlobPool.Datadir = stack.ResolvePath(config.BlobPool.Datadir)
+	}
+	blobPool := blobpool.New(config.BlobPool, eth.blockchain)
+
 	if config.TxPool.Journal != "" {
 		config.TxPool.Journal = stack.ResolvePath(config.TxPool.Journal)
 	}
-	eth.txPool = txpool.NewTxPool(config.TxPool, eth.blockchain.Config(), eth.blockchain)
+	legacyPool := legacypool.New(config.TxPool, eth.blockchain)
 
+	eth.txPool, err = txpool.New(new(big.Int).SetUint64(config.TxPool.PriceLimit), eth.blockchain, []txpool.SubPool{legacyPool, blobPool})
```

Comment:
We shouldn't allow blob txs in the pool. Even if ingress filters them.

Reply:
Hmm, I assumed that blobs won't be accepted before the dencun fork is active? Is that not the case? I can add an explicit filter for the op-stack chains to not accept blobs into the tx pool.

Reply:
you're right. It's asserted by the blobpool while validating transactions. This is fine.